php/commands/scaffold.php

```diff
@@ -403,6 +403,23 @@ private function get_output_path( $assoc_args, $subdir ) {
 	/**
 	 * Generate starter code for a plugin.
 	 *
+	 * ## OVERVIEW
+	 *
+	 * The following files are generated for your plugin by this command:
```

Comment:
It's probably worth noting which files are always generated, vs those that are ignored when `--skip-tests` is included.